osgp/platform/osgp-adapter-domain-core/pom.xml

```diff
@@ -46,6 +46,22 @@
           </nonFilteredFileExtensions>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven.surefire.plugin.version}</version>
+        <dependencies>
+          <dependency>
+              <groupId>org.junit.jupiter</groupId>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)